jobs/JFV3_CYCLONE_GENESIS_JP

```diff
@@ -42,7 +36,7 @@ export SENDDBN=${SENDDBN:-NO}
 ####################################
 # Specify Execution Areas
 ####################################
-export HOMEens_tracker=${HOMEens_tracker:-${NWROOT:?}/ens_tracker.${ens_tracker_ver}}
+export HOMEens_tracker=${HOMEens_tracker:-${PACKAGEROOT:?}/TC_tracker.${TC_tracker_ver}}
```

Comment:
Apologies, I think I misunderstood one of your questions during our chat this week. This can stay as `ens_tracker`, you just needed to add the bit after `PACKAGEROOT` in. This will help with merging with the ens_tracker repo.
```suggestion
export HOMEens_tracker=${HOMEens_tracker:-${PACKAGEROOT:?}/ens_tracker.${ens_tracker_ver}}
```